src/Provider/Doctrine/DoctrineProvider.php

```diff
@@ -189,6 +194,11 @@ public function isAudited(object|string $entity): bool
 
         $entityOptions = $entities[$class];
 
+        if (null === $entityOptions) {
+            // no option defined => $entity is audited
+            return true;
+        }
+
```

Comment:
Early return not really needed as far as I can tell

Reply:
looks like yes, `isset` should handle it